libuavcan/test/util/avl_tree.cpp

```diff
@@ -0,0 +1,611 @@
+/*
+ * Copyright (C) 2019 Theodoros Ntakouris <zarkopafilis@gmail.com>
+ * */
+
+#include <gtest/gtest.h>
+#include <uavcan/util/avl_tree.hpp>
+#include <uavcan/dynamic_memory.hpp>
+#include <uavcan/driver/system_clock.hpp>
+
+using uavcan::AvlTree;
+
+struct Entry{
+    int key;
+    int payload;
+
+    bool operator<(const Entry & other) const
```

Comment:
This file is also full of bad formatting, please fix.

Reply:
I think there is a need for an auto-formatter/linter

Reply:
May I suggest to use this? https://github.com/UAVCAN/libuavcan/pull/152/files

Or maybe this (not quite reliable): https://kb.zubax.com/pages/viewpage.action?pageId=2195703

Reply:
I tried the one from zubax earlier, it had no effect ( only broke my cmake lists). Will try the first one.

Reply:
This will do.